haproxy/src/track_dns

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+if ps ax | grep $0 | grep -v $$ | grep bash | grep -v grep
+then
+   echo "The script is already running."
+   exit 1
+fi
+
+
+if [ -f /etc/environment ]; then
+     . /etc/environment
```

Comment:
Please use `source` instead of `dot` as it's more explicit.

Reply:
Done